src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
 public class Application {
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        int numberOfLottery = 3;
+        Lottery lottery = new Lottery();
+        for (int i = 0; i < numberOfLottery; i++) {
+            System.out.println(lottery.getRandomBall().getColor() + " "
+                    + lottery.getRandomBall().getNumber());
```

Comment:
You should not use get methods, you need to override toString() method and use it.
Get methods can be removed, because we won't use them after the fix

<img width="822" alt="Знімок екрана 2023-01-25 о 20 28 53" src="https://user-images.githubusercontent.com/41166578/214651062-bd471c21-3185-4540-ad98-1f7d0f8d0a1d.png">

```suggestion
            System.out.println(lottery.getRandomBall());
```